packages/esm-patient-clinical-view-app/src/clinical-view-overview/clinical-view-overview.test.tsx

```diff
@@ -44,6 +45,9 @@ describe('ClinicalViewOverview: ', () => {
     mockClinicalViews.map((clinicalView) =>
       expect(screen.getByRole('tab', { name: clinicalView.slot })).toBeInTheDocument(),
     );
+    const breadCrampCheck = screen.getByRole('tab', { name: 'Breadcrumbs' });
```

Comment:
If the assertions you're making are the ones listed above, a preferable way to do this would be to check for the appearance and absence of the two buttons in each case. Like:

```javascript
renderClinicalViewOverview();

expect(screen.getByRole('tablist')).toBeInTheDocument();
expect(screen.getByRole('button', { name: /Add view/i })).toBeInTheDocument();
expect(screen.queryByRole('button', { name: /Edit view/i })).not.toBeInTheDocument();

mockClinicalViews.map((clinicalView) =>
  expect(screen.getByRole('tab', { name: clinicalView.slot })).toBeInTheDocument(),
);

userEvent.click(screen.getByRole('tab', { name: /breadcrumbs/i }));

expect(screen.getByRole('button', { name: /Edit view/i })).toBeInTheDocument();
expect(screen.queryByRole('button', { name: /Add view/i })).not.toBeInTheDocument();
```

Reply:
@denniskigen thanks for the review. issue has been worked on